UKHomeOffice/sgmr-service
# Add "Resend Verification" page when logging in with unverified account

## AC
Given I have completed the create account form
When I try to log in
And I have NOT clicked the activation link from the email
Then I am sent a new activation email
And shown the 'click on your activation link in email' message page

Given I am on the activate/verify your account screen
When I click the resend activation email link
Then I am sent a new activation email
And I am shown a message that a new email has been sent

----

## To test
 - add/update your test email address in GovNotify (Jen can help you set this up if needed)
 - create a new account on sPCR
 - try to log in
   - you should see the activate/verify your account message
   - you should receive a new activation email
 - on the activate/verify your account message page
 - click the resend activation email link
   - you should see a confirmation that a new email has been sent
   - you should receive a new activation email

----

## Notes
